docs/DevelopmentPlan/DevelopmentPlan.tex

```diff
@@ -167,16 +202,93 @@ \section{Expected Technology}
 
 \begin{itemize}
 \item Specific programming language
+\subitem The project can roughly be divided into 3 main components:
+\begin{enumerate}
+  \item A database where the bulk of the site's data will be stored.
+  \item A webserver which will host the website's visual data and code.
+  \item Middleware which will allow for communication between the 
+  webserver and database.
+\end{enumerate}
+\subitem The middleware will be written in Python due to its ease of use and 
+familiarity among team members.
+\subitem We will either use PostgreSQL scripts or an ORM (like SQLalchemy) 
+to implement the database, the language used will depend on the implementation 
+chosen. PostgreSQL scripts would require using only SQL, while an ORM would 
+likely live where with middleware and as such would be written in Python.
```

Comment:
instead of "would likely live where with middleware..." say:
"an ORM would likely live with the middleware..."